lib/features/login/data/network/authentication_client/authentication_client_web.dart

```diff
@@ -74,7 +74,7 @@ class AuthenticationClientWeb implements AuthenticationClientBase {
         scopes: scopes));
 
     if (tokenResponse != null) {
-      final tokenOIDC = tokenResponse.toTokenOIDC();
+      final tokenOIDC = tokenResponse.toTokenOIDC(maybeAvailableRefreshToken: refreshToken);
```

Comment:
What if `refreshToken` is null?

Reply:
if no refreshToken, can not exchange to get newToken, go to login again.